packages/cozy-pouch-link/src/CozyPouchLink.js

```diff
@@ -1,79 +1,49 @@
 import { MutationTypes, CozyLink, getDoctypeFromOperation } from 'cozy-client'
 import PouchDB from 'pouchdb'
 import PouchDBFind from 'pouchdb-find'
-import fromPairs from 'lodash/fromPairs'
 import omit from 'lodash/omit'
+import defaults from 'lodash/defaults'
 
 import { withoutDesignDocuments } from './helpers'
 import { getIndexNameFromFields, getIndexFields } from './mango'
-
+import * as jsonapi from './jsonapi'
+import PouchManager from './PouchManager'
 PouchDB.plugin(PouchDBFind)
 
-const pipe = fn => res => {
-  try {
-    fn(res)
-  } catch (e) {
-    console.warn('Error during pipe', e)
-  }
-  return res
-}
+const sanitized = doc => omit(doc, '_type')
 
-const ensureHasBothIds = obj => {
-  if (!obj._id && obj.id) {
-    obj._id = obj.id
-  }
-  if (!obj._id && obj.id) {
-    obj.id = obj._id
-  }
-  return obj
+const parseMutationResult = (original, res) => {
+  return { ...original, ...omit(res, 'ok') }
 }
 
-const addDoctype = (arr, doctype) => arr.map(x => ({ ...x, _type: doctype }))
+const doNothing = () => {}
```

Comment:
What is it used for?

Reply:
To ensure that the forward option default value is to do nothing